Tests/CSharp/ExpressionTests.cs

```diff
@@ -633,5 +633,62 @@ public string TestMethod()
     }
 }");
         }
+
+
+
+
+
+        [Fact]
+        public void IntegerDivideExpression()
+        {
+            TestConversionVisualBasicToCSharp(@"Class TestClass
+    Sub TestMethod(ByRef i As Integer)
+        Dim k as Integer
+        k = i \ 1000000
+    End Sub
+End Class"
+, @"class TestClass
+{
+    public void TestMethod(ref int i)
+    {
+        int k;
+        k = i / 1000000;
+    }
+}");
+        }
```

Comment:
Yep this is the right sort of test for integer division - now already covered in master by https://github.com/icsharpcode/CodeConverter/blob/4f6bcb3ff7213e321dd73bc3e9e507956e44244e/Tests/CSharp/ExpressionTests.cs#L84